### ISSUE TITLE
Invalid Gemspec Error

### ISSUE THREAD

**Issue by xentek** [kristianmandrup/mongoid-geo#21]
Invalid gemspec in [/Users/username/.rvm/gems/ruby-1.8.7-p299/specifications/mongoid_geo-0.4.1.gemspec]: invalid date format in specification: "2011-04-08 00:00:00.000000000Z"

Getting this after a recent update.

**Comment by stevenkolstad** [comment]
I updated RubyGems from 1.4 to 1.7.2 and received the same error.

**Comment by bastos** [comment]
Me too, same error. You can edit mongoid_geo-0.4.1.gemspec and put the right date (only 2011-04-08), dirty hack but works.

**Comment by xentek** [comment]
After a bit of research it seems that you shouldn't set the date in a Gemspec file. See:  http://rubygems.rubyforge.org/rubygems-update/Gem/Specification.html#method-i-date-3D

So, I forked/cloned the repos from github, ran `gem build mongoid_geo-0.4.1.gem` and `gem install mongoid_geo` from the project directory and now the error seems to have cleared up. I didn't have to change any files (was going to in my fork and send a pull request, etc), so it seems this issue had been addressed, its just not on rubygems.org yet for normal updating.

Anyway, it cleaned up stdout and that's all I cared about. Hope this helps
